bwamem_extra.c declares mem_mark_primary_se() as returning void, but it
returns int. In wasm a call through the wrong signature traps, so declare
it as defined.

diff --git a/bwamem_extra.c b/bwamem_extra.c
index c47b93f..f50cec7 100644
--- a/bwamem_extra.c
+++ b/bwamem_extra.c
@@ -102,7 +102,7 @@ const bwtintv_v *smem_next(smem_i *itr)
 mem_alnreg_v mem_align1(const mem_opt_t *opt, const bwt_t *bwt, const bntseq_t *bns, const uint8_t *pac, int l_seq, const char *seq_)
 { // the difference from mem_align1_core() is that this routine: 1) calls mem_mark_primary_se(); 2) does not modify the input sequence
 	extern mem_alnreg_v mem_align1_core(const mem_opt_t *opt, const bwt_t *bwt, const bntseq_t *bns, const uint8_t *pac, int l_seq, char *seq, void *buf);
-	extern void mem_mark_primary_se(const mem_opt_t *opt, int n, mem_alnreg_t *a, int64_t id);
+	extern int mem_mark_primary_se(const mem_opt_t *opt, int n, mem_alnreg_t *a, int64_t id);
 	mem_alnreg_v ar;
 	char *seq;
 	seq = malloc(l_seq);
